frontend/app/package.json

```diff
@@ -77,6 +78,7 @@
     "@vitest/coverage-v8": "^2.1.2",
     "axios-mock-adapter": "^1.21.2",
     "buffer": "^6.0.3",
+    "chrome-launcher": "^1.1.2",
```

Comment:
Do you know how Chrome gets installed? Is it installed by one of the npm dependencies? Or is it available because of the playwright install in `actions/make_init`?